src/kirby/components/form-field/_form-field-inputs.shared.scss

```diff
@@ -0,0 +1,55 @@
+@import '../../scss/utils';
+
+:host(.error) {
+  input,
+  textarea {
+    border-color: get-color('danger');
+  }
+}
+
+:host(.warning) {
+  input,
+  textarea {
+    border-color: get-color('warning');
+  }
+}
+
+label {
+  display: block;
+  font-size: font-size('s');
+  font-weight: font-weight('light');
+  margin-bottom: size('xxxs');
+  padding: 0 size('s');
+}
+
+input,
+textarea {
+  background-color: get-color('white');
+  border-color: transparent;
+  border-radius: size('s');
+  border-style: solid;
+  border-width: 1px;
+  box-shadow: get-elevation('2');
+  box-sizing: border-box;
+  color: get-color('black');
+  display: block;
+  font-size: font-size('n');
+  line-height: line-height('n');
+  outline: none;
+  padding: size('s');
+  width: 100%;
```

Comment:
Suggestion: 
Add resize: none
Gets rid of that ugly resize arrow - I have no idea why it's not an opt in.